oat-sa/extension-lti-test-review
# Fix/tr 157/remove readonly overlay

Related to: https://oat-sa.atlassian.net/browse/TR-157

With the `DEBUG_MODE` is set to `false`, the readonly overlay is preventing the user from interacting with the items in the reviewer. This overlay is not needed anymore since alternative renderers are used by the review mode.

This PR makes sure this overlay is disabled.

How to test:
- make sure the `DEBUG_MODE` is set to `false`, in `config/generis.conf.php`
- grab an LTI link for a delivery (the QTI Interactions sample is a good candidate since it contains various type of interactions, including extended text and media)
- open the LTI emulator (http://ltiapps.net/test/tc.php), put the link to the delivery into the launch URL, the key, and the secret, then set the role as `Learner`.
- save and launch the activity into another window
- once the test is finished, copy the deliveryExecution parameter (it will be needed to launch the ltiTestReview)
    <img width="1125" alt="image" src="https://user-images.githubusercontent.com/1500098/100768839-a13efe00-33fb-11eb-93bf-a9d2bdc1ce92.png">
- go back to the LTI emulator
- set the URL as `<HOST>/ltiTestReview/ReviewTool/launch?execution=<DELIVERY_EXECUTION>`, replacing the placeholders by your TAO host and the deliveryExecution parameter copied in a previous step
- tick the checkbox `Display optional parameters`
- write the custom parameters: `show_score=1 show_correct=1`
    <img width="1160" alt="image" src="https://user-images.githubusercontent.com/1500098/100769673-ae102180-33fc-11eb-9e9e-e1f41480ee16.png">
- keep the role as `Learner`.
- save and launch the activity into another window
- the ltiTestReview should start with the delivery and the answer previously given
- the interactions should be reachable, but not modifiable. The media player should work without restriction, and the scrollbars should be actionable too
